### PR TITLE
Fixing Pr mistakes

### PR DESCRIPTION
**Pull request by Ollieollieolllie**
fixing pr conflicts that reverted previous fixes

### CODE REVIEW

File: @AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/weapon/fn_weaponGuncontrol.sqf [hunk 1]
```diff
@@ -21,7 +21,11 @@ params ["_heli"];
 _curAcq = [_heli] call fza_fnc_targetingGetSightSelect;
 _curAcq = _curAcq + 1;
 if (_curAcq > 3) then {
-    _curAcq = 0;
+    if !(_heli animationPhase "fcr_enable" == 1) then {
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/weapon/fn_weaponGuncontrol.sqf:24]: You should be able to sight select FCR without an FCR present - think about transmitted targets

File: @AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/targeting/fn_targetingSched.sqf [hunk 2]
```diff
@@ -42,4 +42,7 @@ if (inputAction "vehLockTargets" > 0.5 && fza_ah64_locktargstate == 0 && count _
 };
 if (inputAction "vehLockTargets" < 0.5 && fza_ah64_locktargstate == 1) then {
     fza_ah64_locktargstate = 0;
+};
+if (inputAction "lockTarget" > 0.5) then {
+    fza_ah64_mycurrenttarget = cursorTarget;
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/targeting/fn_targetingSched.sqf:47]: Why is this required? I am trying to keep `fza_ah64_mycurrenttarget` to be just targets selected by the FCR.

File: @AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/ihadss/fn_ihadssDraw.sqf [hunk 3]
```diff
@@ -26,7 +26,7 @@ _zoomstate = 0;
 _targhead = 0;
 
 _sensor = "R ";
-_sensxm = "FCR";
+_sensxm = "TADS";
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/ihadss/fn_ihadssDraw.sqf:29]: Changing this shouldn't make a difference, but it shouldn't break anything either.

File: @AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/event/fn_eventInit.sqf [hunk 4]
```diff
@@ -42,8 +42,8 @@ if !(_heli getVariable ["fza_ah64_aircraftInitialised", false]) then {
     _heli setVariable ["fza_ah64_pfz_count", 0, true];
     _heli setVariable ["fza_ah64_curwpnum", 0, true];
     _heli setVariable ["fza_ah64_waypointdata", [getPos _heli], true];
-    _heli setVariable ["fza_ah64_sight_plt", 0, true];
-    _heli setVariable ["fza_ah64_sight_cpg", 0, true];
+    _heli setVariable ["fza_ah64_sight_plt", 2, true];
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/event/fn_eventInit.sqf:45]: Are you sure you want the default sight select in the pilot's seat to be TADS? I would recommend HMD for both.

**Review by mattysmith22 — CHANGES_REQUESTED** [review]

File: @AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/targeting/fn_targetingSched.sqf (hunk 2 above)
Comment (Ollieollieolllie) [reply @AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/targeting/fn_targetingSched.sqf:47]: nessesery for deselection targets as there is not other method and troubleshooing/ bugfixing with shooting gun through fcr/fcr

**Review by mattysmith22 — APPROVED** [review]
